css-two-column-layout/index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+
+  <div class="container">
+    <div class=" margin-top header-marg text-align-center">
+      <h1 class="col-whole biggest-text">SCALE,SPEED,AGILITY</h1>
+      <p class="width col-6"> We've decided thousands of man hours to making healthcare institutions in the
+    Helium network run 10x more efficiently and deliver an uparalled quality of care.
+      </p>
+    </div>
+    <div class="col-row less-container">
+      <div class="half-columns">
+        <div>
+          <img src="images/software.svg " alt="software">
+        </div>
+        <div>
+          <h3 class="col-4">INTUITIVE, SMART SOFTWARE</h3>
+          <p class="col-6">We build products that transform how healthcare facilities run across Africa
+      - making them radically more efficent operationally and financialy, a complete
+      turnaround.
+          </p>
+        </div>
```

Comment:
This should be one `column-half` element, since it takes up half the width of its row.